### PR TITLE
CAPT-690 Add Review App Link to Pull Request

### PR DESCRIPTION
**Pull request by welshman1970**
Add Githun Action to automatically post the link to the review app when a pull request when a PR is opened.

### CODE REVIEW

File: .github/workflows/ga_publish_ra_link.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+on: pull_request
+
+jobs:
+  example_comment_pr:
+    runs-on: ubuntu-latest
+    name: An example job to comment a PR
+    steps:
+      - name: Checkout
```
Comment (saliceti) [inline .github/workflows/ga_publish_ra_link.yml:8]: Is checkout really required?

File: .github/workflows/ga_publish_ra_link.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+on: pull_request
+
+jobs:
+  example_comment_pr:
```
Comment (saliceti) [inline .github/workflows/ga_publish_ra_link.yml:4]: This is from an example, can it be more meaningful for us

File: .github/workflows/ga_publish_ra_link.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    runs-on: ubuntu-latest
+    name: An example job to comment a PR
```
Comment (saliceti) [inline .github/workflows/ga_publish_ra_link.yml:6]: This is from an example, can it be more meaningful for us

File: .github/workflows/ga_publish_ra_link.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Comment PR
+        uses: thollander/actions-comment-pull-request@v1
```
Comment (saliceti) [inline .github/workflows/ga_publish_ra_link.yml:12]: This adds a comment everytime the job runs. We only really want one. Can we use a sticky comment instead as in https://github.com/DFE-Digital/teaching-vacancies/blob/main/.github/workflows/build_and_deploy.yml#L268

File: .github/workflows/ga_publish_ra_link.yml (hunk 3 above)
Comment (welshman1970) [reply .github/workflows/ga_publish_ra_link.yml:6]: removed

File: .github/workflows/ga_publish_ra_link.yml (hunk 4 above)
Comment (welshman1970) [reply .github/workflows/ga_publish_ra_link.yml:12]: ok i will look into this

**Review by saliceti — APPROVED** [review]

**Pull request merged**